.github/workflows/ci.yml

```diff
@@ -389,3 +389,93 @@ jobs:
             -DWITH_ADIAK=Off \
             $GITHUB_WORKSPACE
             make
+
+  build-rmq-tests:
+    # The type of runner that the job will run on
+    runs-on: ubuntu-latest
+    container: ghcr.io/llnl/ams-ci-almalinux8:latest
+
+    services:
+      rabbitmq:
+        image: rabbitmq:3.11
+        env:
+          RABBITMQ_DEFAULT_USER: guest
+          RABBITMQ_DEFAULT_PASS: guest
+        ports:
+          - 5672
+
+    steps:
+      - uses: actions/checkout@v3
+      - name: Build Torch=On FAISS=On RMQ=On AMS
+        shell: bash -l {0}
+        run: |
+          # module load gcc/11.2.1
```

Comment:
Github test fails (it shouldn't?). Uncomment line

Reply:
I am not sure I understand, that branch is already up to date with `develop`. The file `create-containers.yml` exits in that branch.

Reply:
Yup, you're right. I was mis-selecting `ci-rmq-test` (or something like that) instead of this branch (`ci-rmq-build`). I started the workflow to build the updated the containers.